src/overmapbuffer.cpp

```diff
@@ -464,29 +515,35 @@ std::vector<npc*> overmapbuffer::get_npcs_near_player(int radius)
     return get_npcs_near(plpos.x, plpos.y, plpos.z, radius);
 }
 
-std::vector<overmap *> overmapbuffer::get_overmaps_near( point location, int radius )
+std::vector<overmap*> overmapbuffer::get_overmaps_near( point const location, int const radius )
 {
     // Grab the corners of a square around the target location at distance radius.
     // Convert to overmap coordinates and iterate from the minimum to the maximum.
-    std::set<point> distinct_corners;
```

Comment:
I believe this was an early change I made before I found other ways to work even calling this rather heavy function at all. The rational at the time was that, while both the `set` version and the `vector` versions are O(n・logn), especially for small a small n the `set` version doesn't perform particularly well (poor locality of reference, multiple memory allocations, etc).

I would have to profile things again, but this could potentially be changed back to the set version without affecting performance too much if the vector version seems too roundabout. Better yet, there's likely a smarter algorithm for doing this; I merely transformed the code.